### DISCUSSION TITLE
Changelog strategy

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre-native#371] [Q&A]
How should we maintain changelogs? Currently, we have the following changelog files:

```
./platform/android/CHANGELOG.md
./platform/node/CHANGELOG.md
./platform/macos/CHANGELOG.md
./platform/ios/platform/macos/CHANGELOG.md
./platform/ios/platform/ios/CHANGELOG.md
./platform/ios/CHANGELOG.md
./CHANGELOG.md
```

**Comment by ntadej** [comment]
Android and iOS can stay separate as it should really cover the SDK part. We are anyways discussing if we move SDKs in separate repos in #321 .

I'm not that familiar with Node, but maybe we should sync it with core as it's hard really to split it.

Qt should also be synced with core as it has only a very thin wrapper on top of the core.
